docs/developer/payments/lifecycle.mdx

```diff
@@ -110,6 +138,32 @@ Charge status depends on:
 
 The difference between `total` and `totalGrantedRefund` is compared with **charged** transactions, and the result is one of the statuses: `PARTIAL`, `FULL`, `OVERCHARGED`. If no transaction has been charged, the status is `NONE`.
 
+#### Charge status calculations for Checkout
+
+The status is calculated based on all transactions assigned to the checkout.
+It takes into account the fields: `chargedAmount` and `chargePendingAmount`.
+
+- **NONE**: No funds have been charged.
+- **PARTIAL**: The charged funds do not fully cover the checkout's total amount.
+- **FULL**: The charged funds fully cover the checkout's total. At this point, `checkoutComplete` can be called to convert the checkout into an order.
+However, pending amounts (success/failure) will be confirmed later by the payment app, which means the created order could still remain unpaid if the pending amounts fail.
```

Comment:
We can also add a note that at this point `CHECKOUT_FULLY_PAID` webhook event is triggered